modules/op-setting-a-cron-job-for-refreshing-catalog-in-tekton-hub.adoc

```diff
@@ -0,0 +1,71 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-tekton-hub-with-openshift-pipelines.adoc
+
+:_content-type: PROCEDURE
+[id="setting-a-cron-job-for-refreshing-catalog-in-tekton-hub_{context}"]
+= Setting a cron job for refreshing catalog in {tekton-hub}
+
+[role="_abstract"]
+Cluster administrators can optionally set up a cron job to refresh the database after a fixed interval, so that changes in the catalog reflects in the UI.
+
+[NOTE]
+====
+If new resources are added or existing resources are updated in the catalog, a catalog refresh action displays it in the UI. However, if a resource is deleted in the catalog, a catalog refresh action does not delete the resource in the database, and continues to display it in the UI.
```

Comment:
```suggestion
If new resources are added, or existing resources are updated in the catalog, refreshing the catalog displays these changes in the <specify which UI>. However, if a resource is deleted from the catalog, refreshing the catalog does not remove the resource from the database. The <name of UI> continues displaying the deleted resource.
```

Reply:
@PuneetPunamiya - Rolfe raises a valid point here. Can you please let me know what happens if a user attempts using a resource that has been deleted from the catalog, but still persists in the backend database?